src/lib/coda_base/external_transition.ml

```diff
@@ -63,43 +49,27 @@ end)
   module Protocol_state = Protocol_state
   module Blockchain_state = Protocol_state.Blockchain_state
 
-  type t =
-    { protocol_state: Protocol_state.value
-    ; protocol_state_proof: Proof.Stable.V1.t sexp_opaque
-    ; staged_ledger_diff: Staged_ledger_diff.t }
-  [@@deriving sexp, fields, bin_io]
-
-  module Verified = struct
+  module T = struct
     type t =
       { protocol_state: Protocol_state.value
-      ; protocol_state_proof: Proof.Stable.V1.t
-      ; staged_ledger_diff: Staged_ledger_diff.Verified.t }
+      ; protocol_state_proof: Proof.Stable.V1.t sexp_opaque
+      ; staged_ledger_diff: Staged_ledger_diff.t }
     [@@deriving sexp, fields, bin_io]
 
+    (* TODO: Important for bkase to review *)
```

Comment:
which? The compare?

Reply:
This comment was already on the code. I just moved it. I kept it because I wasn't sure if you saw it on whatever PR is was added to originally. I don't know what it refers to specifically.

Reply:
You can remove it, I imagine I looked at it